### DISCUSSION TITLE
Translation method indicator on fields icon per method

### DISCUSSION THREAD

**Discussion by jripmeester** [craftcms/cms#15741] [Ideas]
Our client noticed that his editor were not fully aware of the different translation methods fields have in their CMS. Some fields are per site and some are per language for example.

They can find this out by hovering over the translation indicator on which a tooltip shows the method but maybe the icon can be per method (in a famliy of icons) so that the method is visually explained.
![translations-of-fields](https://github.com/user-attachments/assets/6691d907-4461-4004-bb6e-90d9ee11191c)

**Comment by richardlandman** [comment]
This can be a BIG UX inprovement for editors of multi-site / multi-language setups (read: enterprise CMS implementations). Currently the various translation settings per field are not obvious. So a better visual contrast between the different options would be great.
